docs/controls/LottieAnimationView.md

```diff
@@ -0,0 +1,150 @@
+
+## Dynamic Properties
+You can update a LottieAnimationView's properties dynamically, at runtime. This can be used for a variety of purposes such as:
+* Theming (day and night or arbitrary themes).
+* Responding to events such as an error or a success.
+* Animating a single part of an animation in response to an event.
+* Responding to view sizes or other values not known at design time.
+
+``You can refer to the original documentation, as Lottie for UWP reimplements all the same functionality (http://airbnb.io/lottie/android/dynamic.html).``
+
+### Usage
+```cs
+animationView.AddValueCallback<ColorFilter>(
+    new KeyPath("Shape Layer", "Rectangle", "Fill"),
+    LottieProperty.ColorFilter,
+    new LottieValueCallback<ColorFilter>(new SimpleColorFilter(Colors.Red)));
+```
+```cs
+animationView.AddValueCallback<Color?>(
+    new KeyPath("Shape Layer", "Rectangle", "Fill"),,
+    LottieProperty.Color,
+    frameInfo =>
+    {
+        return frameInfo.OverallProgress < 0.5 ? Colors.Green : Colors.Red;
+    });
+```
+
+## Syntax
+
+```xaml
+<lottie:LottieAnimationView
+    FileName="Assets/Gears.json"
+    AutoPlay="True"
+    ColorFilter="Transparent"
+    Progress="0.287885218858719"
+    Scale="1"
+    FrameRate="60"
+    Speed="1"
+    RepeatMode="Restart"
+    RepeatCount="-1"
+    MinFrame="0"
+    MaxFrame="52"/>
+```
+
+
+## Example Image
+
+![LottieAnimationView animation](../resources/images/Controls-LottieAnimationView.gif "LottieAnimationView")
+
+## Example Code
+
+[LottieAnimationView Sample Page](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/LottieAnimationView)
+
+The following sample demonstrates how to add a LottieAnimationView Control.
+
+```xaml
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.LottieAnimationViewPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:lottie="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    mc:Ignorable="d">
+
+    <Grid Background="{StaticResource Brush-Grey-05}">
+        <lottie:LottieAnimationView
+            FileName="Assets/Gears.json"
+            AutoPlay="True"
+            ColorFilter="Transparent"
+            Progress="0.287885218858719"
+            Scale="1"
+            FrameRate="60"
+            Speed="1"
+            RepeatMode="Restart"
+            RepeatCount="-1"
+            MinFrame="0"
+            MaxFrame="52"/>
+    </Grid>
+</Page>
+
+
+```
+
+## Requirements (Windows 10 Device Family)
+
+| [Device family](http://go.microsoft.com/fwlink/p/?LinkID=526370) | Universal, 10.0.10586.0 or higher |
+| --- | --- |
+| Namespace | Microsoft.Toolkit.Uwp.UI.Controls.Lottie |
+
+## API
```

Comment:
Good to have a Related Topics section and include links to Adobe resources on the format and tools?

Reply:
Done.